### ISSUE TITLE
The notebook language_modeling.ipynb is not accessible

### ISSUE THREAD

**Issue by piegu** [huggingface/notebooks#125]
Hello,

I can not open the notebook [language_modeling.ipynb](https://github.com/huggingface/notebooks/blob/master/examples/language_modeling.ipynb). 

Instead, the message "An error occurred" is displayed.

cc @sgugger 

Pierre

**Comment by sgugger** [comment]
Yes this notebook is not in the repo anymore. Where did you get the link?

All examples are properly listed in [this doc page](https://huggingface.co/docs/transformers/notebooks)

**Comment by piegu** [comment]
Thanks Sylvain. I got the link in the column **Notebook** of the page https://huggingface.co/docs/transformers/notebooks#pytorch-examples

**Comment by sgugger** [comment]
Ahah, definitely needs an update then! Thanks for flagging, will fix that ASAP.

**Comment by sgugger** [comment]
Ah investigating more, the link is correct but it looks like GitHub has some problem. I can open the notebook locally or using [nbviewer](https://nbviewer.org/github/huggingface/notebooks/blob/master/examples/language_modeling.ipynb).
